### ISSUE TITLE
The application's icon is invisible in dialog windows.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [rmcrackan/Libation#762]
#### **Describe the bug**

As https://github.com/rmcrackan/Libation/issues/760#issue-1912298565 demonstrates with the undermentioned:

![Screenshot_20230925_221305](https://github.com/rmcrackan/Libation/assets/42837531/0410fd7f-1f30-4522-a70a-2ec09911028a) ![Screenshot_20230925_222502](https://github.com/rmcrackan/Libation/assets/42837531/ced254dd-2ce8-4562-8991-e93887e07956)

...unlike main windows, Libation's dialog windows don't display an icon, meaning that left-aligned KWin icons are misaligned.

#### **To Reproduce**

Invoke the tutorial.

#### **Expected behavior**

Either an icon should be present for the application, or the icons should be properly aligned.

#### **Platform**

`cpe:/o:opensuse:tumbleweed:20230922`:

<blockquote>

```py
>>> platform.uname().system
'Linux'
>>> platform.uname().release
'6.5.4-1-default'
>>> platform.uname().version
'#1 SMP PREEMPT_DYNAMIC Wed Sep 20 05:07:04 UTC 2023 (fdd7e9e)'
>>> platform.uname().machine
'x86_64'
```

</blockquote>

#### **Log Files**

> Attach your Libation log file here.

I don't know where to acquire them from.

**Comment by Mbucari** [comment]
This is working as intended. On Windows, message boxes like the one in your first image do not have icons. I styled Chardonnay's message boxes to match classic's styling in this regard. Like it or not, both primary devs (@rmcrackan and myself) are Windows users.

**Comment by beedell-roke** [comment]
> https://github.com/rmcrackan/Libation/issues/762#issuecomment-3133588657

@Mbucari, indeed. In retrospect, this is an example of [`bugs.kde.org/show_bug.cgi?id=501595`][1]. I dare say that I was less knowledgeable about this back in 2023. Thanks.

[1]: https://bugs.kde.org/show_bug.cgi?id=501595#c1
